store/symptom_report.go

```diff
@@ -0,0 +1,80 @@
+package store
+
+import (
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+type SymptomDailyReport struct {
+	Date     string `json:"date"`
+	Symptoms []struct {
+		Name  string `json:"name"`
+		Count int    `json:"count"`
+	} `json:"symptoms"`
+}
+
+func (m *mongoCommunityStore) AddSymptomDailyReports(ctx context.Context, reports []*SymptomDailyReport) error {
```

Comment:
For `reports []*SymptomDailyReport`, I think we do not need an array of pointer here.